Deal.py

```diff
@@ -18,33 +18,29 @@ def pull_random_card(deck):
     card = deck[random.randint(0, 51)]
     return card
 
-def deal(numhands, n, deck=deck):
+def deal(numhands, num_cards, deck=deck):
     #this only will deal with up to 10 players
     current_hands_index = 0
-    working_hand = []
-    hand=[]
-    hand1 = []
-    hand2 = []
+    hands=[]
     pulled_cards = []
     if numhands > 10:
         print("There aren't enough cards for that many people.  Please choose fewer players.")
     else:
         while current_hands_index < numhands:
             current_hands_index = current_hands_index + 1
-            while int(len(working_hand)) < n:
+            working_hand = []
+            while int(len(working_hand)) < num_cards:
                 card = pull_random_card(deck)
                 if card not in pulled_cards:
                     working_hand.append(card)
                     pulled_cards.append(card)
-            else:
```

Comment:
You don't need this else statement because you always want to add the working_hand to the hands list